app/elements/user/challenge/user-challenge-detail.html

```diff
@@ -0,0 +1,110 @@
+<link rel="import" href="/elements/skill/skill-polygon.html">
+
+<dom-module id="user-challenge-detail">
+	<style>
+		:host {
+			display: block;
+			height: 100%;
+			width: 70%;
+			margin-left: 15%;
+			margin-right: 15%;
+		}
+		#userInfoWrapper {
+			height: calc(40% - 24px);
+			margin: 8px 16px 8px 16px;
+			overflow: auto;
+			background: white;
+		}
+		#challengeWrapper {
+			height: 40px;
+			padding-top: 15px;
+			text-align: center;
+			margin: 16px 16px 8px 16px;
+			overflow: auto;
+			background: white;
+		}
+		#challengeWrapper >> h3 {
```

Comment:
[There is no `>>` selector in CSS.](https://developer.mozilla.org/en-US/docs/Web/Guide/CSS/Getting_started/Selectors)